numpyro/hmc_util.py

```diff
@@ -419,7 +419,7 @@ def _double_tree(current_tree, vv_update, kinetic_fn, inverse_mass_matrix, step_
                                         r_ckpts, r_sum_ckpts)
 
     return _combine_tree(current_tree, new_tree, inverse_mass_matrix, going_right, transition_key,
-                         biased_transition=True)
+                         True)
```

Comment:
This change is required for newer version of jax. See https://github.com/google/jax/pull/595 The old version of jax should work fine with this change too.